.github/workflows/python-package-lint.yml

```diff
@@ -30,9 +30,12 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install pylint pylint-quotes
+        python -m pip install mypy types-click types-protobuf types-redis types-requests types-urllib3 typing_extensions
```

Comment:
why u dont want to install the types with mypy automatically ?

Reply:
--install-types with --no-interactive didn't install deps we had. Also not a big fan of letting mypy decide what to install.